Library/Homebrew/formula.rb

```diff
@@ -1622,6 +1622,19 @@ def std_meson_args
     ["--prefix=#{prefix}", "--libdir=#{lib}", "--buildtype=release", "--wrap-mode=nofallback"]
   end
 
+  # Standard parameters for pip builds.
+  sig {
+    params(prefix:          T.nilable(T.any(String, Pathname)),
+           build_isolation: T::Boolean).returns(T::Array[String])
+  }
+  def std_pip_args(prefix: self.prefix, build_isolation: true)
```

Comment:
```suggestion
  def std_pip_args(prefix: prefix, build_isolation: true)
```
I don't think you need `self` here. I also suspect we want `build_isolation` to default to `false` when called directly in `std_pip_args` (but keep it to `true` for `virtualenv_install_with_resources` and friends).

Reply:
tried without `self` but:

```
Library/Homebrew/formula.rb:1630:28: W: Lint/CircularArgumentReference: Circular argument reference - prefix.
  def std_pip_args(prefix: prefix, build_isolation: true)
```

good point about `build_isolation`